### PR TITLE
Pbi 754 ccf openapi

### PR DESCRIPTION
**Pull request by axdanbol**

### CODE REVIEW

File: projects/ccf-api/src/lib/routes/browser.ts [hunk 1]
```diff
@@ -21,7 +21,7 @@ export const browserRoute: RequestHandler = (_req, res, _next) => {
 };
 
 export const openApiRoute: RequestHandler = (_req, res, _next) => {
-  const apiFile = resolve('ccf-api-spec.yaml');
+  const apiFile = resolve('projects/ccf-openapi/spec.yaml');
```
Comment (bherr2) [inline projects/ccf-api/src/lib/routes/browser.ts:24]: We want to use the original, unprocessed spec here. This is for the browser UI.

File: projects/ccf-eui/src/app/core/core.module.ts [hunk 2]
```diff
@@ -13,12 +14,17 @@ import { StoreModule } from './store/store.module';
...
     HttpClientModule,
+
     AnalyticsModule.forRoot({
       gaToken: environment.googleAnalyticsToken,
       appName: 'eui'
     }),
     MousePositionTrackerModule,
 
+    CcfApiModule.forRoot(() => new CcfApiConfiguration({
+      basePath: environment.dbOptions.remoteApiEndpoint
```
Comment (bherr2) [inline projects/ccf-eui/src/app/core/core.module.ts:25]: How is this supposed to work with the webcomponent's remote-api input? Or is that still a work in progress?

File: projects/ccf-api/src/lib/routes/browser.ts (hunk 1 above)
Comment (axdanbol) [reply projects/ccf-api/src/lib/routes/browser.ts:24]: Whoops, forgot to change this when I moved the spec back to top level

File: projects/ccf-eui/src/app/core/core.module.ts (hunk 2 above)
Comment (axdanbol) [reply projects/ccf-eui/src/app/core/core.module.ts:25]: The path is set right before doing the request. This is just the initial value. See: https://github.com/hubmapconsortium/ccf-ui/pull/779/files#diff-a1ceb8e958659afbe8d2580371ced8535cea1197394692ab59cb6f2dbcc75addR188

File: projects/ccf-api/src/lib/routes/v1/utils/parse-filter.ts [hunk 3]
```diff
@@ -95,11 +103,19 @@ function processParameter(result: Filter, key: string, value: unknown): void {
       setIfDefined(result, 'ageRange', parseRange(value, 0, 100));
       break;
 
+    case 'age':
+      setIfDefined(result, 'ageRange', parseMinMaxRange(value, 0, 100));
+      break;
+
     case 'bmirange':
     case 'bmi-range':
       setIfDefined(result, 'bmiRange', parseRange(value, 0, 100));
```
Comment (bherr2) [inline projects/ccf-api/src/lib/routes/v1/utils/parse-filter.ts:112]: I'm fine with value 0, but some have higher max range. Please make sure this aligns with the ranges for bmi and age that is in the portal. (Should also show up in the open api spec I think)

**Review by bherr2 — APPROVED** [review]

**Pull request merged**